src/Microsoft.PowerShell.Commands.Utility/commands/utility/New-Object.cs

```diff
@@ -301,21 +311,30 @@ protected override void BeginProcessing()
                     bool isAllowed = false;
 
                     // If it's a system-wide lockdown, we may allow additional COM types
-                    if (SystemPolicy.GetSystemLockdownPolicy() == SystemEnforcementMode.Enforce)
+                    var systemLockdownPolicy = SystemPolicy.GetSystemLockdownPolicy();
+                    if (systemLockdownPolicy == SystemEnforcementMode.Enforce || systemLockdownPolicy == SystemEnforcementMode.Audit)
```

Comment:
Can this be changed to `if (systemLockdownPolicy != SystemEnforcementMode.None)`?

Reply:
I prefer to be explicit, in case another mode is added in the futre.